Client/mods/deathmatch/logic/CClientPed.cpp

```diff
@@ -514,21 +514,17 @@ void CClientPed::SetPosition(const CVector& vecPosition, bool bResetInterpolatio
         // Don't set the actual position if we're in a vehicle
         if (!GetRealOccupiedVehicle())
         {
-            // Set it only if we're not in a vehicle or not working on getting in/out
-            if (!m_pOccupiedVehicle || GetVehicleInOutState() != VEHICLE_INOUT_GETTING_OUT)
```

Comment:
This check was here, because when vehicle position is updated from puresync it also updates passengers, so it would have warped him to the roof when exiting. Now we receive player puresync when exiting, we need to allow it.